olive/systems/azureml/aml_pass_runner.py

```diff
@@ -21,6 +21,7 @@
 from olive.passes import FullPassConfig, Pass
 from olive.resource_path import create_resource_path
 from olive.systems.utils import get_common_args
+from olive.workflows.run.config import OliveModuleConfig
```

Comment:
did this import introduce circular import?

Reply:
need we move OlivePackageConfig into olive_pass.py or pass_config.py so that the module dependencies between workflow and system could be decoupled?

Reply:
I don't think this is an issue. All tests have passed successfully.

Reply:
I understand there is no issue from the time being. Not sure some body would change the SystemConfig depends on OlivePackageConfig in future. If not, there will no issues. Otherwise, we need avoid the circular imports.

Reply:
Both _olive_pass.py_ and _pass_config.py_ seems like wrong place for `OlivePackageConfig`. How about creating a new config file specifically for this, say _olive/common/olive_config.py_. I could move the `PassModuleConfig` into _pass_config.py_.

Reply:
sure, I am ok with your suggestion.

Reply:
Done!